src/figuresClasses.ts

```diff
@@ -1,19 +1,104 @@
+export enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle'
+}
+
+export type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number
+}
 
+export function roundToHundredths(uglyNumber: number): number {
```

Comment:
if your function returns a primitive type you can remove function return type description
```suggestion
export function roundToHundredths(uglyNumber: number) {
```

Reply:
yes i agree, but linter insisted that I leave the function return type

Reply:
Done